provisioning/main.tf

```diff
@@ -0,0 +1,123 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.27"
+    }
+  }
+
+  required_version = ">= 0.14.9"
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "ap-southeast-1"
+}
+
+resource "aws_instance" "rmf_fullstack" {
+  ami           = "ami-0d058fe428540cd89"
+  instance_type = "c5.2xlarge"
```

Comment:
thanks for the feedback! Just wanted to clarify what you mean from loading from a file? Right now the region is specified in the terraform file itself, for example here: https://github.com/open-rmf/rmf_fullstack_installer/blob/provisioning/provisioning/main.tf#L14

Do you mean to load it from another configuration file? It might be simpler for the user to change the values in here. ( Its a single source of responsibility )

Also, what might a device type be? I'm not too sure :bow:

Reply:
Yep I think that'd be a good idea. By device type I think I meant "instance type". But that can be done later.

Reply:
Alright! In that case, I should have added the super useful suggestions you gave above. I'll go ahead with merging